api/openapi.yaml

```diff
@@ -404,6 +447,15 @@ components:
       required: true
       schema:
         $ref: "#/components/schemas/IdOrSlug"
+    namespace:
+      name: OM-Namespace
+      description: Optional namespace
+      default: default
```

Comment:
Default should be empty, shouldn't it?

Reply:
BTW, we could eliminate the difference in how we handle namespaces and just use `default` as the default similarly to Kubernetes.

Reply:
Scratch that, looks like you reached the same conclusion. 😄